features/pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
   <groupId>org.csstudio</groupId>
   <artifactId>thirdparty-features</artifactId>
-  <version>0.0.3-SNAPSHOT</version>
+  <version>0.0.4-SNAPSHOT</version>
```

Comment:
Not this pom (please revert this one back to 0.0.3-SNAPSHOT)
This pom:
features/org.csstudio.thirdparty.all.feature/pom.xml
Needs to be 1.0.3-SNAPSHOT